src/main/java/it/pagopa/pn/bff/mappers/InfoPaMapper.java

```diff
@@ -0,0 +1,38 @@
+package it.pagopa.pn.bff.mappers;
+
+import it.pagopa.pn.bff.generated.openapi.msclient.external_registries.model.InstitutionResourcePN;
+import it.pagopa.pn.bff.generated.openapi.msclient.external_registries.model.ProductResourcePN;
+import it.pagopa.pn.bff.generated.openapi.server.v1.dto.BffInstitution;
+import it.pagopa.pn.bff.generated.openapi.server.v1.dto.BffInstitutionProduct;
+import org.mapstruct.Mapper;
+import org.mapstruct.Mapping;
+import org.mapstruct.Named;
+import org.mapstruct.factory.Mappers;
+
+import java.util.List;
+
+@Mapper
+public interface InfoPaMapper {
```

Comment:
Rename the mapper. This is the InstitutionProductMapper. Add javadoc as for others mappers